### DISCUSSION TITLE
Does leafmap support other map projections?

### DISCUSSION THREAD

**Discussion by glemoine62** [opengeos/leafmap#746] [Q&A] [answered]
Using other than the default EPSG:3857 projection for leafmap.Map() does not seem to work. This seems to be inherited from either ipyleaflet or folium, which appear to have the same issue.

```
import leafmap
from ipyleaflet.projections import projections

m = leafmap.Map(crs=projections.EPSG4326)

m
```

displays a crooked map (weirdly centered on Russia...) for which the bounds are not correct. Adding any layer on top of this produces non-sense.

What is the correct way to use other map projections?

Guido

**Comment by giswqs** [comment, marked as answer]
See 

https://github.com/jupyter-widgets/ipyleaflet/blob/master/examples/CustomProjections.ipynb
